pom.xml

```diff
@@ -48,14 +48,20 @@
 
         <project.build-time>${timestamp}</project.build-time>
 
-        <wildfly.version>27.0.1.Final</wildfly.version>
-        <wildfly.build-tools.version>1.2.13.Final</wildfly.build-tools.version>
-        <wildfly.core.version>19.0.1.Final</wildfly.core.version>
+        <!-- Upstream WildFly Versions -->
+        <upstream.wildfly.version>29.0.0.Final</upstream.wildfly.version>
```

Comment:
Just a minor note that WildFly 29.0.1.Final was released last week so this could be updated and the Core version below could also be updated to the corresponding 21.1.1.Final version.

Reply:
I did it to match EAP8 and ease alignment process.

Reply:
Ok makes sense, thanks.